requirements.txt

```diff
@@ -1,10 +1,6 @@
-numpy==1.15.2
-pytest==4.0.1
-dill==0.2.8.2
-sklearn==0.0
-theano
-scikit-image==0.14.1
-pymongo==3.7.1
-keras==2.2.4
-PyYAML==3.13
-tensorflow==1.11.0
+numpy>=1.15.2
+pytest>=4.0.1
+dill>=0.2.8.2
+PyYAML>=3.13
+tensorflow>=2.1.0
```

Comment:
I'd lock the TF version -- for instance, TF 2.2 has a different interface for a few functions